### PR TITLE
1090/resources page

### PR DESCRIPTION
**Pull request by jaredcwhite**
## Issue

- Addresses #1090

## Description

This adds a new Additional Resources page, with a link to it in the navbar.

This also adds a few CSS improvements which were missed in previous dev merge.

## How Can This Be Tested/Reviewed?

Go to https://deploy-preview-1181--detroit-public-dev.netlify.app/additional-resources and compare with the source content noted in the issue.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/page_content/AdditionalResources.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+  <InfoCardGrid title="Resources" subtitle="The City of Detroit Housing and Revitalization Department has compiled a list of resources to help you find and maintain your housing.">
```
Comment (emilyjablonski) [inline sites/public/page_content/AdditionalResources.md:1]: What is the reason behind adding this as an `md` file vs in React?

File: sites/public/src/RenderIf.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router"
+
+const RenderIf = (props: { language: string; children: JSX.Element }) => {
+  const router = useRouter()
+
+  if (props.language == "all") {
```
Comment (emilyjablonski) [inline sites/public/src/RenderIf.tsx:6]: Looks like these three ifs can be combined into one if they all return the same thing

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
I spun up https://github.com/CityOfDetroit/bloom/pull/1193 with a few suggested changes, lemme know what you think/if any of the changes you don't agree with

**Comment by jaredcwhite** [comment]
@YazeedLoonat @emilyjablonski Hey I had an idea this morning for a third pass on the issue, so I updated this PR:

* I created local `ResourcesGrid` and `Resource` components, which mirror the components from the `ui-components` packages. This gives us type safety and visibility in case something changes in ui-components down the road.
* By using those local components instead, I can continue authoring the Markdown content using a Markdown text editing workflow.

I also brought over the `RenderIf` refactor.

Let me know what you think! ✌️

**Review by seanmalbert — APPROVED** [review]
Though I knew it was going to happen, I wasn't ready for all the imports. This looks good to me, though we may want to think about having an index in Resources that exports all of the files, so we can do a single destructured import.

**Review by YazeedLoonat — APPROVED** [review]
LGTM

**Pull request merged**